### ISSUE TITLE
TMDB calls failing

### ISSUE THREAD

**Issue by divyanshugrover** [UweTrottmann/SeriesGuide#734]
All calls to TMDB are failing with the following error (obtained from debug mode)

![SmartSelect_20200502-195314_SeriesGuide](https://user-images.githubusercontent.com/8335751/80866897-ea38a380-8cae-11ea-8b2a-89bf3f060a50.jpg)

App version: 54

**Comment by UweTrottmann** [comment]
These exceptions are due to network errors. Try another network connection or wait until the issue is resolved.

Are you in India? TMDB has been blocked by some ISPs there in the past.
https://discuss.seriesgui.de/t/tmdb-sync-show-discover-screen-and-movie-section-may-be-broken-in-india/87?u=uwe

**Comment by divyanshugrover** [comment]
Yes, I am from India. It actually works on the same ISP for most days, but for a couple of days it simply doesn't connect. I'll try reaching out to them to see if they intermittently block TMDB. Thanks.
